app/src/main/java/org/mozilla/focus/popup/PopupUtils.java

```diff
@@ -0,0 +1,96 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.popup;
+
+import android.content.Context;
+import android.content.res.Resources;
+import android.graphics.Color;
+import android.graphics.drawable.Drawable;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.webkit.URLUtil;
+import android.widget.ImageView;
+import android.widget.PopupWindow;
+import android.widget.TextView;
+
+import org.mozilla.focus.R;
+import org.mozilla.focus.session.Session;
+import org.mozilla.focus.utils.DrawableUtils;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+public class PopupUtils {
+
+    public static PopupWindow createSecurityPopup(final Context context, final Session session) {
```

Comment:
\[PMD\] INFO: The method 'createSecurityPopup' has a Cyclomatic Complexity of 10\.